lib/format_parser/version.rb

```diff
@@ -1,3 +1,3 @@
 module FormatParser
-  VERSION = '1.6.0'
+  VERSION = '1.6.1'
```

Comment:
☝️ I'm kind of unclear if this is the right version bump. In a way, changing metric names isn't a breaking change? But it can be considered as such _if_ people make use of the metrics from this gem in their own separate systems, and this causes missing data. I'm not sure if this has the right weight, but please correct me if this needs a more serious bump.

Reply:
Ah fair enough 👍 . Shall I change my PR base-branch to #[kev/prev-144/upgrade-dependencies-drop-eol](https://github.com/WeTransfer/format_parser/tree/kev/prev-144/upgrade-dependencies-drop-eol) or? 

Incl. @Kevin-McGonigle wdyt?